sources/packages/backend/apps/db-migrations/src/sql/Announcements/Create-announcements-table.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE sims.announcements (
+    id SERIAL PRIMARY KEY,
+    message_title VARCHAR(100) NOT NULL,
+    message VARCHAR(200) NOT NULL,
+    start_date TIMESTAMP WITH TIME ZONE NOT NULL,
+    end_date TIMESTAMP WITH TIME ZONE NOT NULL,
+    target text [],
```

Comment:
I missed this in the first round of review. I believe that this column is a `NOT NULL` column. Either way, this and the model file do not match.

<img width="439" alt="image" src="https://github.com/user-attachments/assets/cbaa237e-e2de-46c3-a380-986b540b7977">

Reply:
updated